### DISCUSSION TITLE
Local and Repository Prettier/Format inconsistencies

### DISCUSSION THREAD

**Discussion by Brackets-Coder** [TurboWarp/extensions#1683] [Q&A]
Hello,
I'm encountering an issue. I've submitted a PR for a MIDI extension and I can't get format to fix. I've locally installed the same version of prettier that the repository has and copied the config file from Turbowarp. I've run prettier on my file and updated the PR, but it says format failed. What am I doing wrong?

**Comment by CubesterYT** [comment]
We have prettier rules on this repo, which is defined by the prettier rules file in this repo. We have our own rule choices for styles and whatnot when it comes to formatting.

**Reply by Brackets-Coder** [reply]
I've copied the .prettierrc.json file to my local directory. That's the only configuration file I can find in regards to formatting settings. Anything else I'm missing?

**Reply by CubesterYT** [reply]
Nope, that should be it.

**Reply by Brackets-Coder** [reply]
I'm still confused by this. I have the same prettier version and the same config, and the formatting looks consistent with the rest of the repo, but it's still returning a format error. What am I doing wrong?

**Comment by CST1229** [comment]
also, you don't have to format it yourself, you can just wait for someone to format it if the pr gets close to being merged. but if you do, i think the preferred way is to clone the repository and install node.js if you haven't already (if you use the development server that uses `npm run dev` then you already did those steps), go into the repository folder and do `npm run format`

**Reply by Brackets-Coder** [reply]
I know I don't have to format myself but I wanted to anyway. I didn't want to clone the whole repository and have to do all that as I already have some things set up but I did copy the prettierrc config file and execute npx prettier --write on the file. I just don't understand where the inconsistency is.
